modules/rosa-sts-creating-a-cluster-quickly.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-quickly.adoc
+
+[id="rosa-sts-creating-cluster-using-defaults{context}"]
+= Creating a ROSA cluster with STS using the default options
+
+Through the {product-title} CLI (`rosa`), you can quickly create an OpenShift cluster that uses the AWS Security Token Service (STS). The `--sts` argument enables you to deploy a cluster with STS using the default installation options.
+
+Additionally, you can use `auto` mode to immediately create the required AWS Identity and Access Management (IAM) resources using the current AWS account.
+
+[IMPORTANT]
+====
+Only public and AWS PrivateLink clusters are supported with STS. Regular private clusters (non-PrivateLink) are not available for use with STS.
+====
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest AWS, ROSA, and `oc` CLIs on your installation host.
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installations.
+====
+
+.Procedure
+
+. Create the required account-wide roles and policies, including the Operator policies:
++
+[source,terminal]
+----
+$ rosa create account-roles --mode auto --version 4.8 <1>
+----
+<1> You must specify an OpenShift release by using the `--version` option. The version that is installed when a ROSA cluster is created is determined by the roles and policies that are used.
```

Comment:
@pneedle-rh - Although I know what you mean, maybe we can simplify this to state that the account roles are specific to an OpenShift version and is only created once per account, per cluster version.

Reply:
Thank you both for your comments. The sentence was intended to highlight that version that you specify here for the account roles defines the OpenShift version that is installed when you create the cluster in the next step. The reader might be wondering about how they specify a version to install if they are using the defaults to create the cluster.

@0kashi are there scenarios where you might have multiple account-wide roles and policies of the same OpenShift version? For example, multiple teams might want to install separate clusters in the same AWS account but use separate underlying IAM resources. Can they use a prefix to create a second set of account wide roles and policies that have the same version as an existing set? My understanding from a conversation with @vkareh is that if more than one matching set of account-wide roles are available in your account for a cluster version, an interactive list of options is provided. This PR adds a note about that in the "Creating a cluster with STS using customizations" section.

If that is the case, I could update this line as follows:

`<1> You must specify an OpenShift release by using the `--version` option. The account roles and policies are specific to an OpenShift minor release version.`

Reply:
@bergerhoffer 
> So the version that is installed isn't necessarily the one that you pass in with --version?

When creating each cluster, however, you get to choose which z-stream version to install. But you need to have created account-roles that are compatible with that version, at a y-stream level. So if you want to create a ROSA cluster at 4.8.12, you need to create account roles at 4.8 (4.9 should work too, as they are backwards compatible, but 4.7 won't work)

@0kashi 
> account roles are specific to an OpenShift version and is only created once per account, per cluster version

You can have multiple sets of roles, even for the same version. ROSA will prompt the user to select which ones to use when creating a cluster. However, roles are also backwards compatible (starting in ROSA v1.1.3), so creating a single set of roles for 4.9 would allow the user to install any currently supported version (4.7, 4.8, 4.9)

Reply:
From a conversation with @vkareh, I understand that you can now create a cluster with a lower version than the role policies.

Additionally, the --version option with rosa create account-roles is no longer mandatory. If it is not specified, the latest minor release is assumed.

Also, if you don't specify --version or --interactive when you run rosa create cluster, the installation defaults to the latest stable version. If there are no corresponding roles and policies for the latest version, an error message is produced.

As such, I have updated this call out in both modules to the following:

"<1> You can specify an OpenShift release by using the --version option. The latest stable version is assumed if the option is not included. The account-wide roles and policies are specific to an OpenShift minor release version and are backward compatible."

Also, I have updated step 2 in the quick assembly to state the following:

". Create a cluster with STS using the defaults. When you use the defaults, the latest stable OpenShift version is installed:"

These changes have not been pushed to the fork yet. I will do that once I have worked through the other comments in this PR.